src/DocumentationExamples/CodeExamples/ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.exceptionText.approved.txt

```diff
@@ -1,6 +1,6 @@
 To approve the changes run this command:
-copy /Y "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
+copy /Y "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
```

Comment:
@loneosama same as the other PR, you should revert these paths as they're specific to your system and will fail for other contributors.

Reply:
i will as soon as i am free from work